<!DOCTYPE HTML>
<!--
	Phantom by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Effects of an Introduced Parasitic Nest Fly Philornis downsi on Galapagos Birds</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets2/css/main.css" />
		<noscript><link rel="stylesheet" href="assets2/css/noscript.css" /></noscript>
	</head>
	
				<!-- Main -->
					<div id="main">
						<div class="inner">
							<h1>EASEL (Efficient, Accurate, Scalable Eukaryotic modeLs), a Tool for Improvement of Eukaryotic Genome Annotation</h1>
							<span class="image main"><img src="images/easel.jpg" alt="" /></span>
							<p>•	Development of tool that integrates both RNA folding and functional annotation into a model to enhance gene prediction accuracy<p>
<p>•	Benchmarking of existing annotation across diverse plant genomes (multiexonic vs. monoexonic genes and BUSCO completeness)<p>
<p>•	Optimizing AUGUSTUS and training (with RNA/EST hints) to improve gene prediction for non-model organisms<p>
<p>•	Scaling the framework for organisms with larger genome sizes<p>
							<h1>Expression Analysis of Needle Abscission in Easetern North American Firs</h1>
							<span class="image main"><img src="images/christmastree.png" alt="" /></span>
							<p>•	Assemble RNA-Seq transcript data of three gymnosperm species (Abies balsamea, Abies fraseri and Abies balsamea phanerolepis) using Oyster River Protocol<p>
<p>•	Identify genes expressed at the extremes of needle abscission retention (good vs bad)<p>
							<h1>Complete Genome Assembly of Inga vera</h1>
							<span class="image main"><img src="images/inga vera.jpeg" alt="" /></span>
							<p>•	Using Illumina and PACBIO reads of Inga vera foliar tissue, identify best genome assembler (Flye, SOAP, SPAdes, Shasta, FALCON, Canu, MaSuRCA)<p>
<p>•	Estimate genome size in GenomeScope<p>
		<h1>Comparative Transcriptomic Analysis of Juvenile and Adult Leaf Morphologies in Conifers</h1>
							<span class="image main"><img src="images/Screen%20Shot%202022-08-28%20at%208.21.01%20PM.png" alt="" /></span>
							<p>•	Evaluate differences in anatomy and gene expression patterns of juvenile and adult conifer leaves from two independent lineages (Juniperus flaccida and Pinus cembroides) known to exhibit heteroblastic development<p>
<p>•	Perform de novo RNA-Seq analysis on twelve libraries sequenced on Illumina with HiSeq 2500 (100bp PE)<p>
<p>•	Identify differentially expressed genes in each species using DESeq2 and categorize them by biological functional via network analysis with GeneMANIA<p>
<p>•	Functionally classify Gene Ontology (GO) terms and run OrthoFinder to identify gene families<p>
						</div>
					</div>

		<!-- Scripts -->
			<script src="assets2/js/jquery.min.js"></script>
			<script src="assets2/js/browser.min.js"></script>
			<script src="assets2/js/breakpoints.min.js"></script>
			<script src="assets2/js/util.js"></script>
			<script src="assets2/js/main.js"></script>

	</body>
</html>
